Editor/Completion/XmlCompletionTriggering.cs

```diff
@@ -16,7 +16,7 @@ public static (XmlCompletionTrigger kind, int length) GetTrigger (XmlParser spin
 			bool isExplicit = reason == XmlTriggerReason.Invocation;
 			bool isTypedChar = reason == XmlTriggerReason.TypedChar;
 			bool isBackspace = reason == XmlTriggerReason.Backspace;
-			Debug.Assert (!isTypedChar || typedCharacter == '\0');
+			//Debug.Assert (!isTypedChar || typedCharacter == '\0');
```

Comment:
Why is this commented out?

Reply:
I thought this line is been used for debugging purposes. Also, the experimental instance stops and on continuing, the completion doesn't work. It breaks a lot of stuff. That's why I commented it out.

Reply:
Yes, it's for debugging. If it's breaking, then that's probably because there's a bug.

Reply:
Using this now: 
`if (isTypedChar) {
Debug.Assert (typedCharacter != '\0');
}`